src/transformers/utils/versions.py

```diff
@@ -15,6 +15,7 @@
 Utilities for working with package versions
 """
 
+import importlib.metadata as importlib_metadata
```

Comment:
It might be cleaner to replace everywhere the import from this file to `importlib.metadata` and use `importlib.metadata` instead of `importlib_metadata` (I don't see that many uses so should be quick)

Also getting something in the `meta.yaml` file used for the conda job btw.

Reply:
OK, you are very good!